### ISSUE TITLE
v1 returning non public holidays for getPublicHolidaysForDateRange

### ISSUE THREAD

**Issue by dirkam** [jurajmajer/enrico#12]
Hi,

Is it changed what is returned for getPublicHolidaysForDateRange using v1? It seems that it's returning non public holidays, too.  It used to return only public holidays previously?

Thanks

**Comment by jurajmajer** [comment]
It should work, nothing has changed. Can you clarify what is not working? E.g. using JSON interface this works: https://kayaposoft.com/enrico/json/v1.0/?action=getPublicHolidaysForDateRange&fromDate=04-07-2019&toDate=04-07-2022&country=usa&region=District+Of+Columbia

**Comment by dirkam** [comment]
Can you please check this one as an example?

It seems that it returns non public holidays, too:

https://kayaposoft.com/enrico/json/v1.0/?action=getPublicHolidaysForDateRange&fromDate=04-07-2019&toDate=04-07-2022&country=svk&region=bc

It used to only return public holidays.

**Comment by jurajmajer** [comment]
OK, now I understand the problem. I have fixed it. Please let me know if it is OK for you now.

**Comment by dirkam** [comment]
Thank you, it's OK now.
Was this a bug or do you plan to make this change to v1?

**Comment by jurajmajer** [comment]
I do not plan to change v1. It should stay as is. Until yesterday there was a note saying "not a public holiday" for every holiday which was not a public holiday. But as the method is called getPublicHolidaysForDateRange it makes more sense to return only public holidays. So I excluded observances which are not public holidays.

**Comment by dirkam** [comment]
Thanks for the quick fix.
Something must have changed though, as it used to only return the public holidays, there were no "notes" and observances in the returned data with v1. Don't get me wrong, I'm happy that it's been fixed quickly, just wanted to emphasize this, maybe it reveals some potential other issues.

Thanks for the great service.

**Comment by jurajmajer** [comment]
Originally there were really no "notes" and observances in the returned data with v1. After v2 was created (February 2018) I reconnected v1 interface to v2 datasource. This way if a new country is added to v2 datasource or public holiday is updated it is automatically changed also in v1. This reconnection to v2 datasource caused non public holidays (observances) got included in returned data from v1. I wanted to distinguish them so I added note to them saying "not a public holiday". Fix I have made now is I excluded non public holidays (observances) from returned data from v1 at all.
